doc/source/cdef.rst

```diff
@@ -106,7 +106,7 @@ In order of complexity:
     setup(
         ...,
         setup_requires=["cffi>=1.0.0"],
-        cffi_modules=["package/foo_build.py:ffibuilder"],
+        cffi_ft_modules=["package/foo_build.py:ffibuilder"],
```

Comment:
Shouldn't `setup_requires` and `install_requires` be changed as well here? Though I'm not sure whether docs absolutely need to be updated, since we're not publishing them probably.

Reply:
It's probably worth updating the docs and maybe even publishing them? Publishing them doesn't need to happen for the first release though. I'll update this page.